device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceIO.java

```diff
@@ -136,6 +130,21 @@
         this.state = IotHubConnectionStatus.DISCONNECTED;
     }
 
+    DeviceIO(String hostName, IotHubClientProtocol protocol, SSLContext sslContext, ProxySettings proxySettings, long sendPeriodInMilliseconds, long receivePeriodInMilliseconds)
+    {
+        this.sendPeriodInMilliseconds = sendPeriodInMilliseconds;
+        this.receivePeriodInMilliseconds = receivePeriodInMilliseconds;
+
+        this.state = IotHubConnectionStatus.DISCONNECTED;
+
+        this.transport = new IotHubTransport(hostName, protocol, sslContext, proxySettings, this);
+
+        this.sendPeriodInMilliseconds = sendPeriodInMilliseconds;
```

Comment:
this line and the two below it are repeated in this block - rebase issue?

Reply:
Good catch, it does appear to be a rebase issue